src/services/activityReports.test.js

```diff
@@ -61,15 +61,15 @@ describe('Activity Reports DB service', () => {
     });
 
     it('creates a new report with non-grantee recipient', async () => {
-      const beginningARCount = await ActivityReport.count();
       const report = await createOrUpdate({ ...reportObject, activityRecipientType: 'non-grantee' });
-      const endARCount = await ActivityReport.count();
-      expect(endARCount - beginningARCount).toBe(1);
```

Comment:
Was this some leftover code (perhaps checking that parallel tests weren't creating extra report records)?

Reply:
yep!